src/main/java/frc/robot/subsystems/Carousel.java

```diff
@@ -12,29 +12,59 @@
 import com.revrobotics.ColorSensorV3;
 import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.Robot;
 
 public class Carousel extends SubsystemBase {
     WPI_TalonSRX spinner;
     Encoder carouselEncoder;
     ColorSensorV3 colorSensor = new ColorSensorV3(I2C.Port.kMXP);
     int ballCount = 0;
-
+    // TODO: remove when removing old carousel command
+    public double currentCheckpoint = 0;
+    public double lastCheckpoint = 0;
+    double lastSpinSpeed;
+    public boolean backwards;
```

Comment:
I think we should comment this, that it is public until ShooterCommand is updated.

Reply:
I think Coach Paul was talking about the previous line "public boolean backwards;" In general, variables like this should be private with an accessor to get the value. Making it public allows another class to set the value, something we definitely don't want.

Reply:
should I give it a getter method or leave it public until the shooter is changed?

Reply:
I guess making a getter would be good, and it is not too much code, so easily removed.